pkg/services/oauthtoken/oauth_token_test.go

```diff
@@ -138,20 +148,17 @@ func TestService_TryTokenRefresh_NoRefreshToken(t *testing.T) {
 		Expiry:       time.Now().Add(-time.Hour),
 		TokenType:    "Bearer",
 	}
-	usr := &login.UserAuth{
-		AuthModule:        "oauth_generic_oauth",
-		OAuthAccessToken:  token.AccessToken,
-		OAuthRefreshToken: token.RefreshToken,
-		OAuthExpiry:       token.Expiry,
-		OAuthTokenType:    token.TokenType,
+	usr := &user.SignedInUser{
```

Comment:
This is now a bit misleading, the `token` variable is not actually used anywhere in this test, we rely on the token set by `setupOAuthTokenService`.

Reply:
Changed as requested